0003-I-dapr-compose.md

```diff
@@ -0,0 +1,294 @@
+# Dapr Compose
+
+Author(s): Mukundan Sundararajan
+
+State: Draft
+
+Updated: 16th Nov 2022
+
+## Overview
+
+This is a proposal for `dapr compose` feature to be included in the dapr CLI. This is so that it is easy to orchestrate multiple services that needs to be run in tandem along with their `daprd` sidecars in local self-hosted mode.
+
+## Background
+
+Currently to run multiple services along with `dapr sidecar` locally, users need to run multiple `dapr run` commands, keep track of all ports opened, the components folders that each service refers to, the config file each service refers to etc.
+There are also multiple other flags that can be used to tweak behavior of `dapr run` command eg: `--unix-domain-socket`, `--dapr-internal-grpc-port`, `--app-health-check-path` etc.
+
+This increases the complexity of using dapr in development, where users want to run multiple services in local mode and be able to partially/fully replicate the production sceanrio.
+
+In K8s mode this is alleviated through the use of helm/deployment YAML files. There is currently no such capability available for local self hosted mode.
+
+Asking a user to run multiple different `dapr run` commands each with different flags, increases the complxity for users onboarding onto Dapr.
+
+### Why dapr CLI?
+
+From the initial [proposal](https://github.com/dapr/community/issues/207), the solution was proposed as a seprate repo and CLI in itself. But later it was suggested to use dapr CLI itself to have a `compose` command in it instead of a having a separate CLI.
```

Comment:
using the word `compose` has so many connotation to Docker Compose, it leads to think this is only about running container. Given this is about mostly running processes I would use a different word, such as `dapr start`

Reply:
the only concern that I have here is, we would have too many `start` related commands. 
`dapr init`, `dapr start`, `dapr run`. 

Also `dapr start` does in no way convey that it is a command to start multiple apps in a single go.

Reply:
`dapr up`?